packages/airnode-node/src/evm/templates/template-fetching.ts

```diff
@@ -76,17 +79,50 @@ async function fetchTemplateGroup(
   return [[], templatesById];
 }
 
+export const verifyConfigTemplates = (templateIds: string[], fetchOptions: FetchOptions) => {
```

Comment:
Why do this check here if we want to do it in validator?

Reply:
The difference is that in validator we're checking against the `airnodeAddress` field from the same object, and here we're checking against the state.contracts.AirnodeRrp address which comes from config.chains[] so there could be a mismatch here?

Does this seem redundant? Otherwise how to identify which template is for which chain (to get the correct AirnodeRrp address for the config validator :thinking:

Reply:
Mhhh, but why do you check `airnodeRrpAddress`? I didn't notice that, but the rrp address is the address of the AirnodeRrp protocol, not Airnode.

Reply:
:facepalm: Thanks for catching this, I need to instead be deriving the airnode address from **nodeSettings.airnodeWalletMnemonic** (and this can be done in validator so this verification can be removed entirely).